dprint.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "https://dprint.dev/schemas/v0.json",
+  "projectType": "openSource",
+  "incremental": true,
+  "includes": [
+    "**/*.{md}",
+    "**/*.{json}"
+  ],
+  "plugins": [
+    "https://plugins.dprint.dev/markdown-0.9.2.wasm",
```

Comment:
I guess there is no dprint plugin for ascii doc? 🙈

Reply:
As mentioned, unfortunately not.